src/WebUI/Components/ServiceComponent.scss

```diff
@@ -5,9 +5,25 @@
 
     .s-key {
         color: $secondary-text;
+
+        &.bolt-table-cell {

```

Comment:
.bolt-table-cell --> are we not able to add a class to it.

Reply:
Nidhi & I tried this separately and both of us realized that the Table component isn't fully evolved as of now and so they don't expose a way to apply a custom class to this element so that we can apply a style to it. Unfortunately we have to settle for this now.